### ISSUE TITLE
no data found; multiple error messages.  Installation problem?

### ISSUE THREAD

**Issue by Spidertryer** [smicallef/spiderfoot#1768]
Please provide a description of the issue and any relevant error messages.

Please see error log attached

If you can provide an application stack trace that is even better.

What version of Python are you using? 3.11

What version of SpiderFoot are you using (stable release or Git `master` branch)?  not sure

You may also wish to check if your issue has been posted previously:
[SpiderFoot-B837BD5C.log.csv](https://github.com/smicallef/spiderfoot/files/11115986/SpiderFoot-B837BD5C.log.csv)

* https://github.com/smicallef/spiderfoot/issues

**Comment by bcoles** [comment]
Does your user have permission to write to `/home/<user>/.spiderfoot/spiderfoot.db` ?

**Comment by bcoles** [comment]
Please verify that these files and directories are owned by your user.

```
ls -la $HOME/.spiderfoot
ls -la $HOME/.spiderfoot/spiderfoot.db
```

**Comment by Spidertryer** [comment]
my home folder is greyed out and it doesn't have any permissions info
<img width="552" alt="Screenshot 2023-03-31 at 02 12 05" src="https://user-images.githubusercontent.com/129446085/228998159-c8ef8a6d-fa11-470f-bebf-40fda3a00d51.png">

**Comment by Spidertryer** [comment]
my spider foot folders have these permissions
<img width="285" alt="Screenshot 2023-03-31 at 02 16 18" src="https://user-images.githubusercontent.com/129446085/228998471-c4f91f3a-daf0-49b8-8f43-c6020058432e.png">

**Comment by bcoles** [comment]
Please run these commands in terminal.

```
ls -la $HOME/.spiderfoot
ls -la $HOME/.spiderfoot/spiderfoot.db
```

**Comment by Spidertryer** [comment]
ok here is a screenshot
<img width="618" alt="Screenshot 2023-03-31 at 02 18 40" src="https://user-images.githubusercontent.com/129446085/228998801-e2fd82d4-7018-4e16-86d6-6e72877971a4.png">

**Comment by bcoles** [comment]
Do you have multiple instance of SpiderFoot running?

Do you have multiple versions of SpiderFoot? The logs show SpiderFoot is running from `/Users/test/spiderfoot-4.0/` but  the screenshots above show a directory called `spiderfoot` not `spiderfoot-4.0`.

Are you using SpiderFoot from the GitHub repository (`git clone`) or the downloaded 4.0 release? The 4.0 release is a year old. A GitHub clone is recommended.

On Discord you said you were using SpiderFoot from GitHub. But the logs indicate you're using 4.0.

**Comment by Spidertryer** [comment]
Indeed, I think I had managed to download both.  I have deleted all the spider foot files and have used the git clone.

I get stuck at the pip install as on screen grab; no such file or directory

<img width="570" alt="Screenshot 2023-03-31 at 02 48 11" src="https://user-images.githubusercontent.com/129446085/229002607-a41945d1-cf6c-473d-96fe-66c1fd01846b.png">

**Comment by bcoles** [comment]
Did you delete the directory after cloning it?

**Comment by Spidertryer** [comment]
previously I had 2 copies; one the package and the other the clone.  I have deleted both of these and have now used the git clone command

**Comment by Spidertryer** [comment]
so I believe I now only have one copy; the clone

**Comment by bcoles** [comment]
```
git clone https://github.com/smicallef/spiderfoot
cd spiderfoot
pip install -r requirements.txt
```

It is possible that you have pip installed but it is not aliased to `pip` correctly, in which case try tab-completing `pip`. It might be called something like `pip3` or `pip3.11`.

**Comment by Spidertryer** [comment]
<img width="210" alt="Screenshot 2023-03-31 at 03 38 18" src="https://user-images.githubusercontent.com/129446085/229009006-b8ecea9f-cdc6-461c-aeb2-306198629853.png">

**Comment by Spidertryer** [comment]
![Uploading Screenshot 2023-03-31 at 03.40.03.png…]()

**Comment by bcoles** [comment]
You should now be able to run SpiderFoot.

Ensure SpiderFoot is not already running first; then:

```
./sf.py -l 127.0.0.1:5001
```

I suggest performing a simple scan to ensure everything is working correctly. For example, scan `spiderfoot.net` with only the `DNS Resolver` module enabled. You can do this be selecting `New Scan`, entering `spiderfoot.net`, clicking `By Module`, deselecting all modules and selecting only `DNS Resolver`. Check the terminal output for errors.

**Comment by Spidertryer** [comment]
Its working.

Thank you so much for your help.
